content/en/deployment/strategies/overview.md

```diff
@@ -8,106 +8,102 @@ categories: ["Deployment Strategies", "Features", "Concepts"]
-- runWebhook - Allows you to execute tests and scripts using your existing tooling, paired with `exposeServices` you can leverage this step to execute automated tests against the 'green' environment.  
+1. A new "green" environment running your changes is created without any traffic being routed to it
+1. You expose the "green" environment internally using the CD-as-a-Service [preview feature]({{<ref "reference/deployment/config-preview-link">}}). 
+1. You execute your test plan against the "green" environment 
+1. Once your testing and approval requirements are met traffic is shifted to the "green"
+1. You monitor the "green" environment until a set of requirements is fulfilled (such as manual approvals, canary analysis, or automated testing passes)
+1. Once your requirements are fulfilled the "blue" environment is shut down and the leaving only the new version running.
 
-### Benefits of the Blue-Green Deployment Strategy
-The blue-green deployment strategy gives you the benefits of running tests against a full copy of your production environment 
-prior to any traffic being routed to the new version. This allows for zero-downtime deployments and reduced risk when exposing
-users to new versions of your application. 
+### Benefits of the blue/green deployment strategy
+The blue/green deployment strategy gives you the benefits of running tests against a full copy of your production environment prior to any traffic being routed to the new version. This gives you for zero downtime deployments and reduced risk when exposing users to new versions of your app. 
 
-In addition, because the blue-green strategy maintains the 'blue' environment until the shutdown conditions are met it 
-gives you the ability to instantly roll back to a known stable version should any issues be detected during the deployment. 
+In addition, because the blue/green strategy maintains the 'blue' environment until the shutdown conditions are met it, blue/green gives you the ability to instantly roll back to a known stable version should you detect any issues during the deployment. 
 
 ## When to use each strategy
 
-Both the canary and blue-green strategies have trade-offs that can help you determine which strategy is best for your use case. 
+Both the canary and blue/green strategies have trade-offs that can help you determine which strategy is best for your use case. 
 
-### When to use the Canary strategy
+### When to use the canary strategy
 
 Canary is best for use cases where you want to minimize risk by exposing your changes to a small percentage of users and 
 monitor prior to exposing the changes to the majority user base. This allows you to ensure stability and performance of 
 the new version before creating a widespread impact. This minimizes the impact of potential issues on user.
```

Comment:
Extra blurb to consider adding - 

To use a canary strategy effectively, the performance of the canary version needs to be accurately measured. Additionally, it is necessary to determine if the performance of the canary falls within an acceptable range. For instance, a common metric used is the percentage of 5xx responses in the canary version. While integrating monitoring tools to conduct canary analysis is straightforward, finding the correct thresholds for health metrics can be challenging. A canary strategy is suited for you, if you are already aware of metrics and thresholds that determine the health of your application.